addons/sourcemod/translations/server_redirect.phrases.txt

```diff
@@ -3,103 +3,122 @@
 	"command_spam_attempt"
 	{
 		"en"		"[Server Redirect] Please wait a few seconds before you can use this command again."
+		"ru"		"[Редирект] Подождите несколько секунд перед повторным использованием команды."
 	}
 
 	"servers_menu_title"
 	{
 		"en"		"Servers list:"
+		"ru"		"Список серверов:"
 	}
 	
 	"servers_menu_no_servers"
 	{
 		"en"		"No servers available"
+		"ru"		"Нет доступных серверов"
 	}
 	
 	"servers_menu_server_entry"
 	{
 		"#format"	"{1:s},{2:s}"
 		"en"		"{1} {2}"
+		"ru"		"{1} {2}"
 	}
 	
 	"servers_menu_server_entry_no_socket"
 	{
 		"#format"	"{1:s}"
 		"en"		"{1}"
+		"ru"		"{1}"
 	}
 	
 	"servers_menu_server_entry_slots_count"
 	{
 		"#format"	"{1:i},{2:i}"
 		"en"		"[{1}/{2}]"
+		"ru"		"[{1}/{2}]"
 	}
 	
 	"servers_menu_server_entry_not_available"
 	{
 		"en"		"[N/A]"
+		"ru"        "[Н/Д]"
 	}
 	
 	"servinfo_menu_title_no_socket"
 	{
 		"#format"	"{1:s},{2:s}"
 		"en"		"{1}\nIP: {2}"
+		"ru"		"{1}\nIP: {2}"
 	}
 	
 	"servinfo_menu_title"
 	{
 		"#format"	"{1:s},{2:s},{3:s}"
 		"en"		"{1}\nIP: {2}\nMap: {3}"
+		"ru"		"{1}\nIP: {2}\nКарта: {3}"
 	}
 	
 	"servinfo_menu_title_no_player_info"
 	{
 		"#format"	"{1:s},{2:s},{3:s},{4:s}"
 		"en"		"{1}\nIP: {2}\nMap: {3}\nOnline: {4}"
+		"ru"		"{1}\nIP: {2}\nКарта: {3}\nОнлайн: {4}"
 	}
 	
 	"servinfo_menu_map_not_available"
 	{
 		"en"		"N/A"
+		"ru"		"Н/Д"
 	}
 	
 	"servinfo_menu_connect"
 	{
 		"en"		"Connect"
+        	"ru"		"Подключиться"
 	}
 	
 	"servinfo_menu_connect_current"
 	{
 		"en"		"[You are on it]"
+        	"ru"		"[Вы играете на]"
```

Comment:
This doesn't sounds right to me, are you sure about correctness of this? This is shown like ``Connect [You are on it]`` and only when you are trying to connect to server you are currently playing on.